deprecations.md

```diff
@@ -0,0 +1,48 @@
+# Deprecations
+
+## Upcoming
+
+### 31st March 2021
+
+#### Deprecated operations
+
+Following legacy operations will be completely removed:
```

Comment:
do we also mention somewhere that they're consolidated into the all-purpose /getAll endpoints? pre-emptively answer reader's next question

Reply:
I will add reference to what operation is being replaced with